[[ Bug 11133 ]] Update release notes for noui uninstaller

livecodesebastien opened the pull request.

livecodesebastien committed:
[[ Bug 11133 ]] Update release notes for noui uninstaller

peter-b commented on docs/notes/setup.md:
```diff
@@ -56,6 +56,13 @@ On Windows, you need to do:
 	�<installerexe>/Contents/MacOS/installer� install noui *options*
 On both platforms, the result of the installation will be written to the console.
+# Command-line uninstallation
+It is possible to uninstall LiveCode from the command-line on Windows and Linux, from LiveCode *6.7.8* and *7.1.1*. When invoked in this fashion, no GUI will be displayed.
```
I don't think you need to include the explicit version here -- the fact that this information is in a particular version's release notes implies that the behaviour is available for that version.

livecodesebastien committed:
[[ Bug 11133 ]] Remove version reference for command-line uninstaller

Remove the explicit reference of LiveCode 6.7.8 and LiveCode 7.1.1 as being the first versions with the noui uninstaller

peter-b commented on docs/notes/setup.md:
```diff
@@ -56,6 +56,13 @@ On Windows, you need to do:
+	`<exe> uninstall noui`
+Where <exe> is *.setup.exe* on Windows, and *.setup.x86* on Linux. This executable, for both of the platforms, is located in the folder where LiveCode got installed.
+Same as for the installer, there is no authentication possible with the command-line variant of the uninstaller. It means that you need to run the uninstaller as administrator to uninstall LiveCode, if it was installed in an admin-only location.
+The result of the installation will be written in the console.
```
I think it might be more idiomatic English to write,

> The results of the installation will be written to the console.

livecodesebastien committed:
[[ Bug 11133 ]] Fix sentence in command-line uninstallation, in setup.md

livecodeali commented on docs/notes/setup.md:
```diff
@@ -56,6 +56,13 @@ On Windows, you need to do:
+It is possible to uninstall LiveCode from the command-line on Windows and Linux. When invoked in this fashion, no GUI will be displayed.
+On both platforms, the command is of the following form:
+	`<exe> uninstall noui`
+Where <exe> is *.setup.exe* on Windows, and *.setup.x86* on Linux. This executable, for both of the platforms, is located in the folder where LiveCode got installed.
```
Can you change this to "where LiveCode is installed" perhaps?

livecodeali commented on docs/notes/setup.md:
```diff
@@ -56,6 +56,13 @@ On Windows, you need to do:
+On both platforms, the command is of the following form:
+	`<exe> uninstall noui`
+Where <exe> is *.setup.exe* on Windows, and *.setup.x86* on Linux. This executable, for both of the platforms, is located in the folder where LiveCode got installed.
+Same as for the installer, there is no authentication possible with the command-line variant of the uninstaller. It means that you need to run the uninstaller as administrator to uninstall LiveCode, if it was installed in an admin-only location.
```
Maybe "In the same way as for the installer"?

livecodesebastien committed:
[[ Bug 11133 ]] Fix few other mistakes in setup.md

livecodeali commented:
@livecode-vulcan review ok 814847d

livecode-vulcan commented:
💙 review by @livecodeali ok 814847d61370178186920842fe83366b3d22cfd3

livecode-vulcan commented:
😎 test success 814847d61370178186920842fe83366b3d22cfd3
- try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/528)
- try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/532)
- try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/527)
- try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/527)
- try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/536)
- try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/530)
- try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/538)
